keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -3013,10 +3013,10 @@ GA:
 
 - [x] API annotations properly support strategic merge patch.
 - [x] Design and implement borrowing between priority levels.
-- [ ] Satisfaction with LIST and WATCH support.
-- [ ] Implement borrowing by exempt priority levels.
-- [ ] APF allows us to disable client-side rate limiting without causing the apiservers to wedge/crash.  Note that there is another level of concern that APF does not attempt to address, which is mismatch between the throughput that various controllers can sustain.
-- [ ] Satisfaction that the interface is sufficient to support tuning of capacity and resource costs.
+- [x] Satisfaction that LIST and WATCH support are good enough.
+- [x] Implement borrowing by exempt priority levels.
+- [x] APF allows us to disable client-side rate limiting without causing the apiservers to wedge/crash.  Note that this falls a bit short of saying that we would actually want to disable client-side rate limiting.  While experiments show that APF can prevent server crashes, they do not show that the SLOs are met in all scenarios.  If the control plane is under-provisioned then the performance will have to suffer in one way or another.
+- [x] Satisfaction that the interface has sufficient tunability to be useful.
```

Comment:
Given you're touching it, let's also update kep.yaml:
https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1040-priority-and-fairness/kep.yaml

to change the milestone for stable to 1.29

With that, I think the KEP will be ready to be tracked for release 1.29.